ActiveField.php

```diff
@@ -364,15 +370,9 @@ protected function createLayoutConfig($instanceConfig)
 
         if ($layout === 'horizontal') {
             $config['template'] = "{label}\n{beginWrapper}\n{input}\n{error}\n{endWrapper}\n{hint}";
-            $cssClasses = [
-                'offset' => 'col-sm-offset-3',
-                'label' => 'col-sm-3',
-                'wrapper' => 'col-sm-6',
-                'error' => '',
-                'hint' => 'col-sm-3',
-            ];
+            $cssClasses = $this->horizontalCssClasses;
             if (isset($instanceConfig['horizontalCssClasses'])) {
-                $cssClasses = ArrayHelper::merge($cssClasses, $instanceConfig['horizontalCssClasses']);
+                ArrayHelper::merge($this->horizontalCssClasses, $instanceConfig['horizontalCssClasses']);
```

Comment:
this statement is a bit confusing:
1. I would expect the output of ArrayHelper to be used here
2. Previously you assigned $cssClasses. I would expect this here